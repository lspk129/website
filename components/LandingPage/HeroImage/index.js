import styled from 'styled-components';

export default styled.div`
  background-image: url("http://trendytheme.net/demo2/wp/69/multipage/wp-content/uploads/revslider/web-product-dark/webproduct_darkbg2.jpg");
  background-position: center center;
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  margin: -10px;
  height: 100vh
`;

