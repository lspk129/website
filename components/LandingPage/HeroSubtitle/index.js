import React from 'react';
import styled from 'styled-components';

const HeroSubtitle = styled.section`
    text-align: center;
    position: absolute;
    top: 55%;
    left: 50%;
    transform: translate(-50%, -50%);
    color: white;
    font-family: sans-serif;
`;


export default () => (
  <HeroSubtitle>
    <h2>Creating fast & interactive web applications with React and Meteor</h2>
  </HeroSubtitle>
);
