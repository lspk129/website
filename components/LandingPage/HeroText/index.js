import React from 'react';
import styled from 'styled-components';

const HeroText = styled.section`
    text-align: center;
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    color: white;
    font-size: 40px;
    font-family: sans-serif;
    width: 480px
`;

export default () => (
  <HeroText>
    <h1>Moka Haiku: web app agency</h1>
  </HeroText>
);
