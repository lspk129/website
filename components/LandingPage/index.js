import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.section`
  background-image: url("http://trendytheme.net/demo2/wp/69/multipage/wp-content/uploads/revslider/web-product-dark/webproduct_darkbg2.jpg");
  background-position: center center;
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  height: 100vh
`;

export default () => (
  <div>
    <Wrapper />
  </div>
);
