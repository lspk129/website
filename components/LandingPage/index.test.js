import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';


import LandingPage from './index';

describe('LandingPage component', () => {
  it('renders styled-component Wrapper', () => {
    const component = renderer.create(<LandingPage />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Wrapper component', () => {
  it('renders style class with image', () => {
    const component = shallow(<LandingPage />);
    expect(component.find('div').length).toEqual(1);
  });
});
