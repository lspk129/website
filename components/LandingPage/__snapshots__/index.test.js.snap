// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders styled-component Wrapper 1`] = `
<div>
  <section
    className="kHXxIQ"
  />
</div>
`;
